src/nanovg_mtl.h

```diff
@@ -101,6 +101,8 @@ void* mnvgImageHandle(NVGcontext* ctx, int image);
 void mnvgReadPixels(NVGcontext* ctx, int image, int x, int y, int width,
                     int height, void* data);
 
+int mnvgCreateImageFromHandle(NVGcontext* ctx, void* textureId, int imageFlags);
```

Comment:
Please move the prototype to the position between `mnvgCommandQueue()` and `mnvgDevice()`.

Reply:
done, also added a comment, please check you like it